include/Symbol.h

```diff
@@ -25,7 +25,9 @@ enum class SymbolKind {
     Attribute,
     Genvar,
     GenerateBlock,
-    LocalVariable
+    LocalVariable,
+    ProceduralBlock,
+    Data
```

Comment:
I think I would just call this is a Variable symbol instead of data, and LocalVariable will probably be removed in favor of that.